### PR TITLE
Fixing Nits and the incorrect URL for beta-1 release notes

### PR DESCRIPTION
**Pull request by iamsamirzon**
Signed-off-by: Author Name <iamsamir@amazon.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ At this time, the Notary community is working on Notary v2 specification  and tw
...
 *The following milestones lead up to the first release. During alpha and beta milestones, backwards compatibility is not guaranteed.  Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). Test cases will be added along the way. After RC-1 backward compatibility will be maintained**
-- [alpha-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-1) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-1.MD) : Goal - Get feedback on the notation CLI client and notation user experience
+- [alpha-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-1) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-1.MD): Goal - Get feedback on the notation CLI client and notation user experience
 - [alpha-2](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-2) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-2.MD): Goal - Merge open PR on the Notaryv2 standard, CLI and Notation specifications;
```
Comment (yizha1) [inline README.md:19]: nit: remove `;` at the end for consistency.

File: README.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ At this time, the Notary community is working on Notary v2 specification  and tw
...
-- [alpha-3](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-3) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-3.MD):  Goal - Introduce Plugins for signing and verification; Manage signatures in OCI registries using [ORAS Artifacts Specification v1.0.0-RC.2] (https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2)
+- [alpha-3](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-3) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-3.MD): Goal - Introduce Plugins for signing and verification; Manage signatures in OCI registries using [ORAS Artifacts Specification v1.0.0-RC.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2)
 - [alpha-4](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3Aalpha-4) [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.alpha-4.MD): Goal - Add COSE signature format;
```
Comment (yizha1) [inline README.md:21]: nit: remove `;` at the end for consistency.

File: README.md [hunk 3]
```diff
@@ -15,13 +15,13 @@ At this time, the Notary community is working on Notary v2 specification  and tw
...
-- [RC-2](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-2)       [ ] Goal - Support "Fallback" method for OCI 1.0 based registries to manage signatures. 
+- [beta-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ABeta-1)   [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.beta-1.MD): Goal - Verify using Trust Store and Trust Policy 
+- [RC-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-1)       [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.rc-1.MD)): Goal -First supported release from Notary v2 community. JWS and COSE signature formats. Notation CLI client. 
+- [RC-2](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-2)       [TBD] Goal - Support "Fallback" method for OCI 1.0 based registries to manage signatures.
```
Comment (yizha1) [inline README.md:24]: nit: remove `.` at the end for consistency.

File: README.md [hunk 4]
```diff
@@ -15,13 +15,13 @@ At this time, the Notary community is working on Notary v2 specification  and tw
...
-- [RC-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-1)       [Release Notes](https://github.com/notaryproject/roadmap/blob/52c48cdd1937f63bb84a9d02942f5e3ebd6dc909/RELEASENOTES/v2.0.0.rc-1.MD)): Goal -First supported release from Notary v2 community. JWS and COSE signature formats. Notation CLI client. 
-- [RC-2](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-2)       [ ] Goal - Support "Fallback" method for OCI 1.0 based registries to manage signatures. 
+- [beta-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ABeta-1)   [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.beta-1.MD): Goal - Verify using Trust Store and Trust Policy 
+- [RC-1](https://github.com/iamsamirzon/roadmap/projects/1?card_filter_query=milestone%3ARC-1)       [Release Notes](https://github.com/notaryproject/roadmap/blob/main/RELEASENOTES/v2.0.0.rc-1.MD)): Goal -First supported release from Notary v2 community. JWS and COSE signature formats. Notation CLI client.
```
Comment (yizha1) [inline README.md:23]: nit: remove `.` at the end for consistency.

File: README.md [hunk 5]
```diff
@@ -15,13 +15,13 @@ At this time, the Notary community is working on Notary v2 specification  and tw
 
 ### Leading up to the first stable release
 *The following milestones lead up to the first release. During alpha and beta milestones, backwards compatibility is not guaranteed.  Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). Test cases will be added along the way. After RC-1 backward compatibility will be maintained**
```
Comment (yizha1) [inline README.md:17]: nit: remove redundant `*` at the end.

Comment (iamsamirzon) [reply README.md:17]: done

**Pull request merged**